packages/@o3r/rules-engine/README.md

```diff
@@ -59,6 +74,13 @@ Otter framework provides a set of code generators based on [angular schematics](
 | facts-service             | Generate a facts service                                      | `ng g facts-service`             |
 | operators                 | Generate an operator                                          | `ng g operator`                  |
 
-## Details
+## Debug
+A whole section of the [Otter Chrome Devtool extension](https://chromewebstore.google.com/detail/otter-devtools/aejabgendbpckkdnjaphhlifbhepmbne) is dedicated to the debug of the rules engine with
+a visual representation of your Rulesets and a history of the run, conditions met and actions applied.
+
+Find more information on the tool in the [Otter Chrome Devtool extension documentation](https://github.com/AmadeusITGroup/otter/blob/main/docs/dev-tools/chrome-devtools.md).
```

Comment:
I think we could add a link to the `how-it-works/chrome-extension.md` here too, because it is explaining very well the dedicated section of rules engine inside the chrome extension